packages/shorebird_cli/lib/src/commands/patch_command.dart

```diff
@@ -248,47 +205,101 @@ Please create a release using "shorebird release" and try again.
     final downloadReleaseArtifactProgress = logger.progress(
-      'Downloading release artifact',
+      'Downloading release artifacts',
     );
-    try {
-      releaseArtifactPath = await _downloadReleaseArtifact(
-        Uri.parse(releaseArtifact.url),
+    for (final releaseArtifact in releaseArtifacts.entries) {
+      try {
+        final releaseArtifactPath = await _downloadReleaseArtifact(
+          Uri.parse(releaseArtifact.value.url),
+        );
+        releaseArtifactPaths[releaseArtifact.key] = releaseArtifactPath;
+      } catch (error) {
+        downloadReleaseArtifactProgress.fail('$error');
+        return ExitCode.software.code;
+      }
+    }
+    downloadReleaseArtifactProgress.complete();
+
+    final patchArtifactBundles = <Arch, PatchArtifactBundle>{};
+    final createDiffProgress = logger.progress('Creating artifacts');
+
+    for (final releaseArtifactPath in releaseArtifactPaths.entries) {
+      final archMetadata =
+          ShorebirdBuildMixin.architectures[releaseArtifactPath.key]!;
+      final patchArtifactPath = p.join(
+        Directory.current.path,
+        'build',
+        'app',
+        'intermediates',
+        'stripped_native_libs',
+        'release',
+        'out',
+        'lib',
+        archMetadata.path,
+        'libapp.so',
       );
-      downloadReleaseArtifactProgress.complete();
-    } catch (error) {
-      downloadReleaseArtifactProgress.fail('$error');
-      return ExitCode.software.code;
+      final patchArtifact = File(patchArtifactPath);
+      final hash = _hashFn(await patchArtifact.readAsBytes());
+      try {
+        final diffPath = await _createDiff(
+          releaseArtifactPath: releaseArtifactPath.value,
+          patchArtifactPath: patchArtifactPath,
+        );
+        patchArtifactBundles[releaseArtifactPath.key] = PatchArtifactBundle(
+          arch: archMetadata.arch,
+          path: diffPath,
+          hash: hash,
+        );
+      } catch (error) {
+        createDiffProgress.fail('$error');
+        return ExitCode.software.code;
+      }
     }
+    createDiffProgress.complete();
 
-    final String diffPath;
-    final createDiffProgress = logger.progress('Creating diff');
-    try {
-      diffPath = await _createDiff(
-        releaseArtifactPath: releaseArtifactPath,
-        patchArtifactPath: patchArtifactPath,
-      );
-      createDiffProgress.complete();
-    } catch (error) {
-      createDiffProgress.fail('$error');
-      return ExitCode.software.code;
+    final archNames = patchArtifactBundles.keys.map((arch) => arch.name);
+
+    logger.info(
+      '''
+
+${styleBold.wrap(lightGreen.wrap('🚀 Ready to publish a new patch!'))}
+
+📱 App: ${lightCyan.wrap(app.displayName)} ${lightCyan.wrap('(${app.id})')}
+📦 Release Version: ${lightCyan.wrap(releaseVersion)}
+📺 Channel: ${lightCyan.wrap(channelArg)}
+🕹️  Platform: ${lightCyan.wrap(platform)} ${lightCyan.wrap('(${archNames.join(', ')})')}
```

Comment:
```suggestion
🕹️ Platform: ${lightCyan.wrap(platform)} ${lightCyan.wrap('(${archNames.join(', ')})')}
```

Reply:
that's intentional because of how they are rendered 😆

Reply:
Oh lol. We need monospace emojis 😄